app/grandchallenge/reader_studies/views.py

```diff
@@ -1300,15 +1311,23 @@ def new_interface_url(self):
     def process_data_for_object(self, data):
         """Updates the display set"""
         instance = self.object
-        for ci_slug, new_value in data.items():
-            if ci_slug == "order":
+        non_civ_data_keys = ("order", "title")
+
+        save = False
+        for key in non_civ_data_keys:
+            value = data.get(key, empty)
```

Comment:
Why not keep with the default (return `None` if the key is not in the data dictionary)?

Reply:
This is future-proof: it accounts for nullable fields. Generally, whenever doing data retrieval it's better to reply on `empty` to detect it's no-presence rather than `None`.